table/queries/merged_reduced_scans.sql

```diff
@@ -58,22 +67,25 @@ WITH AllScans AS (
     WHERE NOT controls_failed
     GROUP BY date, source, country_code, network, outcome, domain, category, subnetwork
     # Filter it here so that we don't need to load the outcome to apply the report filtering on every filter.
-    HAVING NOT STARTS_WITH(outcome, "setup/")
+    HAVING NOT STARTS_WITH(outcome, "❔setup/")
 )
 SELECT
     Grouped.* EXCEPT (country_code),
     IFNULL(country_name, country_code) AS country_name,
     CASE
-        WHEN (STARTS_WITH(outcome, "expected/")) THEN 0
-        WHEN (STARTS_WITH(outcome, "dial/") OR STARTS_WITH(outcome, "setup/") OR ENDS_WITH(outcome, "/invalid")) THEN NULL
-        WHEN (ENDS_WITH(outcome, "unknown")) THEN count / 2.0
-        ELSE count
+        WHEN (STARTS_WITH(outcome, "✅")) THEN 0
+        WHEN (STARTS_WITH(outcome, "❗️")) THEN count
+        WHEN (STARTS_WITH(outcome, "❔")) THEN NULL
```

Comment:
the only remaining ❔ outcome that is in the final table is now `❔unknown`.

I've switched that to NULL instead of count/2 on the grounds that count/2 never made any sense. unknown is by definition an error in our data handling.

Reply:
unknown was for errors we don't know how to process.
I worry that we will be ignoring things we shouldn't.
The goal of making them 1/2 was to draw attention to them. Because anything that is not ~0 or ~100% needs further investigation, so using 1/2 pulls the results to more investigation, rather than hiding them one way or another.
Perhaps we need to take a look if we have any more unknowns in the data and where they are coming from.

Reply:
Perhaps you can use ± or 🤷 for unknown.

Reply:
Currently unknown is the only outcome type that will have the ❔ emoji, since setup is already entirely filtered out of the dashboard. That's what will help us notice it. Having fractions in the unexpected counts will not help because in practice many of them are already fractional.

There are currently only 2 measurements marked as unknown in the entire hyperquack dataset going back to 2018. [data](https://docs.google.com/spreadsheets/d/1hLTH83D-7QQXgpE0hLNi4E7_S5GaCUGllsCbfZq63-M/edit#gid=1809681870)

❔ is a reasonable way to differentiate this case. The other symbols seem overly silly.